dNet/AuthPackets.cpp

```diff
@@ -29,16 +29,16 @@ void AuthPackets::HandleHandshake(dServer* server, Packet* packet) {
 	inStream.Read(clientVersion);
 
 	server->GetLogger()->Log("AuthPackets", "Received client version: %i", clientVersion);
-	SendHandshake(server, packet->systemAddress, server->GetIP(), server->GetPort());
+	SendHandshake(server, packet->systemAddress, server->GetIP(), server->GetPort(), server->GetServerType());
 }
 
-void AuthPackets::SendHandshake(dServer* server, const SystemAddress& sysAddr, const std::string& nextServerIP, uint16_t nextServerPort) {
+void AuthPackets::SendHandshake(dServer* server, const SystemAddress& sysAddr, const std::string& nextServerIP, uint16_t nextServerPort, const ServerType serverType) {
```

Comment:
We could pass serverType by reference here instead of copying it over - just a small nit